### DISCUSSION TITLE
Do you guys think that the laravel model is very slow?

### DISCUSSION THREAD

**Discussion by allowing** [laravel/framework#38001] [Q&A]
![image](https://user-images.githubusercontent.com/20262949/125571634-ebbe9367-cbdf-4a6f-aecc-d39e812a8e77.png)
↑↑ Using model  (With model)

![image](https://user-images.githubusercontent.com/20262949/125572045-550269ae-c47b-4e9f-8208-8fa88bb80e2e.png)
↑↑ Using stdClass  (Without model)

![image](https://user-images.githubusercontent.com/20262949/125571969-7fcc8ac6-75c2-4b8a-9a15-c96d515d8857.png)

**Comment by allowing** [comment]
If you have tens of thousands of cycles, it is not recommended to use the model in the cycle. 😁

**Reply by kohenkatz** [reply]
No matter what technology you choose, there will always be some places which are better-suited for it than others. That is not unique to Eloquent Models.

Using Models provides you with all kinds of features, but of course those features come at a cost.

**Comment by AhemdHegazy** [comment]
instead of this 
use DB Laravel query builder in query the database , use eloquent model in creating ,updating and deleting
